docs/reference/resources/coap-attributes-request.sh

```diff
@@ -2,9 +2,9 @@
 # for ThingsBoard Cloud
 
 # Send CoAP attributes request
-coap get coap://coap.thingsboard.cloud/api/v1/$ACCESS_TOKEN/attributes?clientKeys=attribute1,attribute2&sharedKeys=shared1,shared2
+coap-client -m get "coap://coap.thingsboard.cloud/api/v1/$ACCESS_TOKEN/attributes?clientKeys=attribute1,attribute2&sharedKeys=shared1,shared2"
```

Comment:
This is the only script where we should change "coap" to "coap-client". All other scripts should use "coap" from https://www.npmjs.com/package/coap-cli because it is easier to install on windows.

Reply:
@ashvayka done.